lib/1_is_greater_than_five.rb

```diff
@@ -5,5 +5,9 @@
 # Run `bin/check` from the command line to execute the automated tests.
 
 def is_greater_than_five(num)
-  # Write your code here.
+  if num > 5
+      return true
+    else
```

Comment:
Nice! Watch indentation here. `else` should be in line with `if` and both of the `return` statements should be in line with one another.